ui/app/templates/jobs/run/templates/manage.hbs

```diff
@@ -0,0 +1,34 @@
+{{page-title "Manage templates"}}
+<section class="section">
+    <header class="run-job-header">
+        <h1 class="title is-3">Choose from a template</h1>
+        <p>A plan will be requested before the job is submitted.  You can pick a template or write your own job spec.</p>
+    </header>
+    {{#if (eq this.model.length 0)}}
+        <h3 data-test-empty-templates-list-headline class="empty-message-headline">
+          You have no templates to choose from. Would you like to <Hds::Link::Inline @route="jobs.run.templates.new" data-test-create-inline>create</Hds::Link::Inline> one?
+        </h3>
+        <Hds::Button class="button-group" @text="Back to editor" @route="jobs.run" @icon="arrow-left" data-test-cancel />
+    {{else}}
+    <main class="radio-group" data-test-template-list>
+        <Hds::Table @model={{this.model}} @columns={{ create-columns "Name" "Namespace" "Description"}}>
+            <:body as |B|>
```

Comment:
Could we change `B` to `Template` or something more indicative of what we're iterating over?

Reply:
This isn't a template. Rather it's a [contextual components](https://guides.emberjs.com/v4.9.0/components/block-content/) that is yielding out `TableRow` and `TableData` components. We could change the name to `Body` to reflect this. And its a pattern used by [other teams](https://github.com/hashicorp/boundary-ui/blob/a6b2acc14d0de00ff5b8aa2a3568f1d57db99393/ui/admin/app/components/form/target/details/index.hbs#L17), in our own codebase and in line with the [official documentation](https://helios.hashicorp.design/components/table?tab=code).

Changing this to `Body` could make sense. But that would defeat the purpose of the `alias` and we could just write `:body.Tr` instead.

Your call. I'm open to suggestions!

Reply:
This makes a ton of sense. Thanks for clearing it up for me!